pkg/config/config.go

```diff
@@ -214,12 +219,16 @@ func NewConfig(confString string, c *cli.Context) (*Config, error) {
 	}
 
 	if conf.Limit.NumTracks == 0 {
```

Comment:
Should we turn this on by default? I'm concerned that it will surprise users who are simply performing an upgrade, and then finding that participants are being rejected before actual limits have been reached.

Reply:
My idea was to set default limits high enough where they wouldn't be noticeable. I'll add a flag to enable enforcement of these limits, with default set to false.